target/linux/ramips/dts/ZBT-WE2026.dts

```diff
@@ -0,0 +1,102 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zbtlink,zbt-we2026", "ralink,mt7620n-soc";
+	model = "Zbtlink ZBT-WE2026";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "zbt-we2026:red:power";
+			gpios = <&gpio1 14 1>;
+		};
+
+		air {
+			label = "zbt-we2026:green:air";
```

Comment:
According to the [Hardware specifications](http://www.zbtlink.com/products/router/ZBT-WE2026.html) this LED is named WLAN.